### PR TITLE
Improve timeline page animation

### PR DESCRIPTION
**Pull request by ks129**
~~I know that this didn't get approved in #dev-contrib, but when I read timeline, this disturb me more and more.~~

Now when animation go out of screen, this animates hiding process. This also allows bounce in again.

Video: https://www.dropbox.com/s/oq7v23fs8idwpjv/PyDis%20Timeline.mov?dl=0

### CODE REVIEW

File: pydis_site/static/js/timeline/main.js [hunk 1]
```diff
@@ -32,17 +32,36 @@
...
 		}
 	};
 
+	VerticalTimeline.prototype.hideBlocksScroll = function () {
+		if ( ! "classList" in document.documentElement ) {
+			return;
+		}
+		var self = this;
+		for( var i = 0; i < this.blocks.length; i++) {
+			(function(i){
```
Comment (lemonsaurus) [inline pydis_site/static/js/timeline/main.js:54]: I'd probably nitpick a little on the weird floaty var and exclamation mark - but I guess this is the prevailing style in this file, so I suppose it's fine.

**Review by lemonsaurus — APPROVED** [review]
You know what, this looks pretty good, and it's a harmless improvement that you clearly feel strongly about.

I appreciate you including a video. Approved.

File: pydis_site/static/js/timeline/main.js (hunk 1 above)
Comment (ks129) [reply pydis_site/static/js/timeline/main.js:54]: I copy-pasted this part.

**Comment by jb3** [comment]
Neat work!

**Review by jb3 — APPROVED** [review]

**Pull request merged**